crates/store/re_chunk/src/transport.rs

```diff
@@ -731,7 +731,7 @@ mod tests {
                 {
                     chunk_roundtrip = &chunk_in_transport;
                 }
-                let chunk_after = Chunk::from_transport(&chunk_roundtrip)?;
+                let chunk_after = Chunk::from_transport(chunk_roundtrip)?;
```

Comment:
unrelated to this pr, but showed up whenever I did `clippy --all-targets`

Reply:
it breaks CI lol

Reply:
okay this should be fixed forever now....